codeclimate/workflows.yaml

```diff
@@ -13,14 +13,19 @@ workflows:
   codeclimate/add_private_repo:
     description: Add a private Github repository to Code Climate
     with:
+      org_id: |-
+        {{- with .ctx.org_id | trim }}
+        {{- if empty . }}{{ fail "Please specify a Code Climate organization ID" }}{{ end -}}
+        {{ . }}
+        {{- end }}
       org: |-
-        {{- with .ctx.org | trim | trimPrefix "@" }}
-        {{- if empty . }}{{ fail "Please specify a Github organization" }}{{ end -}}
+        {{- with .ctx.repo | trim | trimPrefix "https://github.com/" | splitList "/" | first | trimPrefix "@" }}
+        {{- if empty . }}{{ fail "Please specify a Github organization name" }}{{ end -}}
```

Comment:
So, this workflow only takes `repo` as input. No `org`, right?

Reply:
That's correct. I am splitting the `$ctx.repo` into `org` and `repo` here